misc/git-hooks/pre-commit

```diff
@@ -59,7 +59,7 @@ function check_check_forbidden_dependencies() {
       >&2 echo "fedimint-server/Cargo.toml must not depend on modules"
       return 1
     fi
-    if grep -E "(fedimint-mint|fedimint-wallet|fedimint-ln-(server|common|client))" fedimint-testing/Cargo.toml >&2 ; then
+    if grep -E "(fedimint-mint|fedimint-wallet|fedimint-ln-(server|client))" fedimint-testing/Cargo.toml >&2 ; then
```

Comment:
@m1sterc001guy @joschisan Why it was there and why is it OK now? :D

Reply:
Fedimint testing needs the PrunedInvoice now.